qadence_libs/qinfo_tools/qng.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import annotations
+
+from typing import Callable
+
+import torch
+from qadence import QuantumCircuit
+from torch.optim.optimizer import Optimizer, required
+
+from qadence_libs.qinfo_tools.qfi import get_quantum_fisher, get_quantum_fisher_spsa
+
+
+class QNG(Optimizer):
+    """Implements the Quantum Natural Gradient Algorithm."""
+
+    def __init__(
+        self,
+        params: tuple | torch.Tensor,
+        circuit: QuantumCircuit = required,
```

Comment:
Is this default value intrinsic to `Optimizer` ?

Reply:
[Yes](https://github.com/pytorch/pytorch/blob/main/torch/optim/optimizer.py#L49C1-L49C6)